research/object_detection/g3doc/installation.md

```diff
@@ -21,10 +21,8 @@ instructions](https://www.tensorflow.org/install/). A typical user can install
 Tensorflow using one of the following commands:
 
 ``` bash
-# For CPU
-pip install tensorflow
-# For GPU
-pip install tensorflow-gpu
+# For CPU and GPU
+pip install tensorflow==1.15
```

Comment:
For TensorFlow 1.x, CPU and GPU packages are still separate:
tensorflow==1.15 —Release for CPU-only
tensorflow-gpu==1.15 —Release with GPU support (Ubuntu and Windows)

https://www.tensorflow.org/install/pip